app/models/recipe.rb

```diff
@@ -0,0 +1,13 @@
+class Recipe < ApplicationRecord
+  belongs_to :user
+  has_many :recipe_foods
```

Comment:
- [ ] When we try to delete a Recipe with recipe_foods referencing it, the app is crashing with the following error
![Screenshot from 2023-05-21 07-59-36](https://github.com/chiztechnology/rails-capstone-project/assets/56886064/68c6bc64-5a6c-4d0f-a65f-05b218b292cf)
The fix is pretty the same as the fix you will apply on food